software/utils/rpi_config/headless_pi.md

```diff
@@ -0,0 +1,61 @@
+# Remote configuration via ssh 
+
+You may want to configure a Raspberry Pi without a dedicated keyboard and mouse.
+
+Herein we will refer to your local development machine as `Local` and the remotely accessed Raspberry Pi as `PI`.
+
+1. Flash the operating system to your SD card.
+2. Mount the SD card on your local machine.
+3. Copy the 2 files in the [boot](boot) directory to the `boot` volume on the SD card.
+4. Modify the [wpa_supplicant.conf](boot/wpa_supplicant.conf) file with the correct [country code](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) and the WiFi `network name` and `wifi password`.
+5. Insert the SD card into a Raspberry Pi (a standalone will do, does not have to be a fully integrated ventilator).
+6. On your `local` machine, use bash:
+```shell
+ssh-keygen -R raspberrypi.local
+ssh pi@raspberrypi.local
+```
+where the password will be `raspberry`. The first command may throw an error, but that is ok. It is just to clear any old credentials.
+
+7. Now that you are on the `PI`, run `sudo raspi-config` and set the machine's hostname to be `rw-ventilator` or something more unique if you expect to have multiple of them on your network simultaneously. When exiting the configuration utility, don't reboot it just yet.
+8. Also on the `PI`:
```

Comment:
That is a lot of steps which only serve the purpose of renaming the user. Would it be that bad to keep things simple and keep the `pi` username?

Alternately, could we provide some level of automation to these steps?

Reply:
Using `rpi-imager` to flash the image takes care of this for us.